invenio_jobs/services/schema.py

```diff
@@ -138,6 +138,8 @@ def get_data_type(self, data):
         data_type = super().get_data_type(data)
         if data_type is None:
             return "custom"
+        else:
+            return data_type
```

Comment:
@palkerecsenyi that is the fix for the since validation?

Reply:
But you still need to change the custom schema to pass the job_args_schema f.e. in the `ror-http` task?